extend/custom-theme.md

```diff
@@ -104,3 +104,109 @@ app.Settings().SetTheme(&myTheme{})
 
 With these changes you can apply your own style, to make small tweaks or
 provide a completely custom looking application!
+
+### Workable Example
```

Comment:
I looked at the other tutorials and it seems to me like they don’t add a new section just for the code. We probably should stay consistent.
```suggestion
In the end, the resulting code could look something like this:
```